wordpress-mobile/WordPress-iOS
# Reader: Update no results view for Reader blog follows

Closes #22403 
Merges into: #22406

## Description

Updates the no results view when the user has no blog subscriptions.

## Screenshots

| Light | Dark |
|---|---|
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-01-18 at 12 08 59](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/2412a291-4fc5-4a8c-b2ab-a4354b9b7926) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-01-18 at 12 09 02](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/7f6cec65-6eee-43ab-ae92-e56bca0afd82) |

## Testing

To test:
- Set up an account with no blog subscriptions
- Launch Jetpack and login
- Navigate to the `Subscriptions` screen
- 🔎 **Verify** the UI matches the new UI
- Tap on `Discover Blogs`
- 🔎 **Verify** it navigates to the `Discover` feed

## Regression Notes
1. Potential unintended areas of impact
Other screens which use the `NoResultsViewController`

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A
PR submission checklist:


UI Changes testing checklist:
